package.json

```diff
@@ -10,12 +10,17 @@
   },
   "scripts": {
     "lint": "semistandard --fix",
+    "styles": "curl -o viewer/viewer/static/main.css https://www.consumerfinance.gov/static/css/main.css && curl https://www.consumerfinance.gov/static/apps/regulations3k/css/main.css >> viewer/viewer/static/main.css",
```

Comment:
This is my new favorite CSS build tool for micro-sites. It's called "copy/paste and use the same patterns as cf.gov".